yjit/src/codegen.rs

```diff
@@ -5305,6 +5305,7 @@ fn gen_send_general(
                 );
             }
             VM_METHOD_TYPE_ATTRSET => {
+                #[allow(clippy::if_same_then_else)]
```

Comment:
```suggestion
                #[cfg_attr(
                    not(feature = "stats"),
                    allow(clippy::if_same_then_else)
                )]
```
If you run `cargo clippy --all-features` this goes away. Without all the stats features, these are identical branches. We can be a bit more specific with this and only allow when stats are not enabled.

Just fyi @noahgibbs

Reply:
Ah, that makes sense! Thank you.